components/raftstore-v2/src/worker/pd/store.rs

```diff
@@ -231,6 +231,8 @@ where
         stats.set_cpu_usages(self.store_stat.store_cpu_usages.clone().into());
         stats.set_read_io_rates(self.store_stat.store_read_io_rates.clone().into());
         stats.set_write_io_rates(self.store_stat.store_write_io_rates.clone().into());
+        // Update grpc server status
+        stats.set_is_grpc_paused(self.grpc_service_manager.is_paused());
```

Comment:
How is this information (is_grpc_paused) used in PD?

Reply:
For:

By design, PD will notify the slow TiKV node to restart Grpc if necessary(). And this flag is a flag reflecting whether the Grpc server has been paused or not in the target node.

This strategy is not open until the detection on transient jitters has been merged.